pkg/operator/ceph/operator.go

```diff
@@ -169,6 +170,11 @@ func (o *Operator) Run() error {
 			close(stopChan)
 			o.clusterController.StopWatch()
 			return nil
+		case err := <-mgrErrorChan:
+			logger.Errorf("gave up to run the operator. %v", err)
+			close(stopChan)
+			o.clusterController.StopWatch()
+			return nil
```

Comment:
Let's return the err here so that the caller will call `TerminateFatal()` and the error will be visible in the pod description.

Reply:
Fixed, thanks.